Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.8
+
+ENV PYTHONUNBUFFERED 1
+
+WORKDIR /discoreg
+EXPOSE 80
```

Comment:
We'll be running this behind nginx, so can we use 8000 (or whatever gunicorn's default port is) here?